change/@fluentui-react-dialog-ce4d9fc2-0bea-4c0a-9683-8e86082cd6ad.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33471&commitId=76b7ace334850f27f28611a697eba7a83786e3a8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|4| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|28| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|2819| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>